src/Components/Header/Header.scss

```diff
@@ -27,46 +29,65 @@
 
 .zip-code-par {
   margin: 0;
+  margin-top: 20px;
   padding: 5px;
-  border: 3px solid $dark_green_border_color;
-  color: white;
+  font-size: 1.2em;
+  font-weight: 600;
+  letter-spacing: 0.5px;
+  color: cornsilk;
   border-radius: 10px;
-  width: 75%;
+  width: 90%;
   min-width: 125px;
   text-align: center;
 }
 
 .back-button {
   width: 80%;
+  color: cornsilk;
+  font-weight: 500;
+  border-bottom-color: cornsilk;
+  background-color: transparent;
+  letter-spacing: 0.5px;
   min-width: 145px;
   padding: 5px;
   font-size: 1em;
-  border-radius: 10px;
-  border: 0;
+  // border-radius: 20px;
+  // border: 0;
+  border-top: none;
+  border-right: none;
+  border-left: none;
   outline: 0;
   cursor: pointer;
-  border: 3px solid $dark_green_border_color;
 }
 
 .back-button:hover,
 .back-button:focus {
-  background-color: $list_hover_color;
+  // background-color: $list_hover_color;
+  border: 2px solid khaki;
+  border-radius: 20px;
+  // font-weight: 00;
+  // color: black;
+  // border: none;
   transform: scale(1.05);
 }
 
 .vendor-login-button {
+  background-color: transparent;
```

Comment:
I feel like we need some sort of background on this button to help identify that it is a button - its harder to tell with it just being text